content/chapter-01/LabStarAsAService.md

```diff
@@ -0,0 +1,23 @@
+# Lab: IaaS, PaaS, CaaS, FaaS, and DaaS
+
+* Do you have a account with your cloud provider?
+  * If you do, login into the cloud provider console interface
+  * If you do not have an account, create one right now. It should take from five to fifting minutes to do so
+
+
+* On your console, look around, and see what they offer
+
+
+* Which of their services are:
+  * Compute, e.g.: Virtual Machines, Containers, Functions/Lambdas
+  * Storage, e.g.: block storage, RDMS, NoSql
```

Comment:
I assume you meant RDBMS rather than RDMS.
I would separate Storage and Database - I feel there is a clear conceptual difference